Tasks/SqlAzureDacpacDeployment/FindSqlPackagePath.ps1

```diff
@@ -92,6 +92,36 @@ function Get-RegistryValueIgnoreError
     return $null
 }
 
+function Get-RegistrySubKeysIgnoreError
```

Comment:
In our task based L0 tests, we have mocked Get-SqlPackageOnTargetMachine method itself. So L0 tests will not be affected by the change. In extension, I have updated L0 test corresponding to this change.